### PR TITLE
353 pause scheduler

### PR DESCRIPTION
**Pull request by tkosgrabar**
## Brief, plain english overview of your changes here
Naive implementation of paused state

## Fixes

#353 

## Reminders
- [ ] Added/ran automated tests
- [ ] Update README and/or examples

---
cc @kagkarlsson

### CODE REVIEW

File: db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/Scheduler.java [hunk 1]
```diff
@@ -35,8 +29,21 @@
...
+import com.github.kagkarlsson.scheduler.task.Execution;
+import com.github.kagkarlsson.scheduler.task.ExecutionComplete;
+import com.github.kagkarlsson.scheduler.task.ExecutionOperations;
+import com.github.kagkarlsson.scheduler.task.OnStartup;
+import com.github.kagkarlsson.scheduler.task.SchedulableInstance;
+import com.github.kagkarlsson.scheduler.task.Task;
+import com.github.kagkarlsson.scheduler.task.TaskInstance;
+import com.github.kagkarlsson.scheduler.task.TaskInstanceId;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```
Comment (kagkarlsson) [inline db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/Scheduler.java:46]: Would rather avoid this import diff..

**Comment by kagkarlsson** [comment]
Looks ok on first glance

File: db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/Scheduler.java (hunk 1 above)
Comment (tkosgrabar) [reply db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/Scheduler.java]: done

**Review by kagkarlsson — APPROVED** [review]
Looks good 👍

**Comment by tkosgrabar** [comment]
nice, when do you plan to merge and release it ?

**Pull request merged**